json-rpc/src/stream_rpc/startup.rs

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) The Diem Core Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::stream_rpc::transport::websocket::get_websocket_routes;
+use diem_config::config::StreamConfig;
+use std::sync::Arc;
+use storage_interface::DbReader;
+use warp::{filters::BoxedFilter, Filter, Reply};
+
+/// Gets all routes for all streaming endpoints
+/// Each transport is responsible for handling it's own versioning and endpoints
+/// This function also handles disabling/enabling streaming- returning a 404 if not enabled
+pub fn get_stream_routes(
```

Comment:
any reason that this isn't in the only use-case file?

Reply:
this structure makes more sense when there was SSE as well. I could move it to the websockets transport file but then I'd have to move it back here when adding SSE. Does that make sense?

Reply:
cool